### PR TITLE
Cardiff Airport Brand

### PR DESCRIPTION
**Pull request by andrew-c**
#### What does this PR do? (please provide any background)

Start of the Cardiff Airport Brand.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/cardiff-airport/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@holidayextras/brand-cardiffairport",
+  "version": "1.0.0",
```
Comment (elliottcrush) [inline brands/cardiff-airport/package.json]: Can we bump this to 1.0.1? I think we've already published 1.0.0

File: brands/cardiff-airport/src/less/type.less [hunk 2]
```diff
@@ -0,0 +1,88 @@
+@import '../../../brand-globals/src/less/type';
+
+//== Font
+//
+
+@font-face {
+  font-family:'Blake Regular';
+  src:url('https://scdn.thomascook.com/oneweb/fonts/fs-blake-regular.eot');
```
Comment (elliottcrush) [inline brands/cardiff-airport/src/less/type.less]: Copy paste fail

File: brands/cardiff-airport/package.json (hunk 1 above)
Comment (jodiedoubleday) [inline brands/cardiff-airport/package.json]: 1.0.0 release? what happened to 0.0.1?

Comment (elliottcrush) [reply brands/cardiff-airport/package.json]: That was my fault. I did the initial release on 1.0.0

File: brands/cardiff-airport/src/less/buttons.less [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+// --------------------------------------------------
+
+.btn-primary {
+  border: none;
+  text-transform: uppercase;
+  font-weight: 700;
+  &:hover {
+    background-color: @brand-primary-dark;
+  }
+  .fa {
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/buttons.less]: if never showing font awesome, should we be including it in package.json?

File: brands/cardiff-airport/src/less/jumbotron.less [hunk 4]
```diff
@@ -0,0 +1,58 @@
+@import '../../../brand-globals/src/less/jumbotron';
+
+//
+// Jumbotron - NEEDS A BIG TIDY
+// --------------------------------------------------
+
+.jumbotron {
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/jumbotron.less]: has this been copied from HX brand? is this required?

File: brands/cardiff-airport/src/less/mixins.less [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+// --------------------------------------------------
+
+// Components
+@import (reference) "mixins/list_group";
+
+.font-awesome {
+  font-family : fontawesome;
+}
+
+.hx-font {
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/mixins.less]: I'm not sure this is needed

File: brands/cardiff-airport/src/less/navbar.less [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  > li > a {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  a {
+    text-decoration: none;
+  }
+}
+.navbar-text {
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/navbar.less]: **Suggestion** there needs to be a distinction here between `navbar-default` and `navbar-inverse` both wont require white text

File: brands/cardiff-airport/src/less/scaffolding.less [hunk 7]
```diff
@@ -0,0 +1,12 @@
+@import '../../../brand-globals/src/less/scaffolding';
+
+//
+// Scaffolding
+// --------------------------------------------------
+
+
+// Temporary to help debugging
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/scaffolding.less]: delete?

File: brands/cardiff-airport/src/less/sub-navs.less [hunk 8]
```diff
@@ -0,0 +1,25 @@
+//
+// Sub-Navs
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/sub-navs.less]: not sure any of this file is required.

File: brands/cardiff-airport/src/less/theme.less [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+// Components
+@import 'component-animations';
+@import 'dropdowns';
+@import 'button-groups';
+@import 'input-groups';
+@import 'navs';
+@import 'navbar';
+@import 'breadcrumbs';
+@import 'pagination';
+@import 'pager';
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/theme.less]: can delete

File: brands/cardiff-airport/src/less/theme.less [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+// Components
+@import 'component-animations';
+@import 'dropdowns';
+@import 'button-groups';
+@import 'input-groups';
+@import 'navs';
+@import 'navbar';
+@import 'breadcrumbs';
+@import 'pagination';
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/theme.less]: can delete

File: brands/cardiff-airport/src/less/theme.less [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+// Core CSS
+@import 'blocks';
+@import 'backgrounds';
+
+// Components
+@import 'lists';
+@import 'progress-trackers';
+
+// Components w/ JavaScript
+@import 'owl-carousel';
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/theme.less]: *may* be able to delete

File: brands/cardiff-airport/src/less/wells.less [hunk 12]
```diff
@@ -0,0 +1,22 @@
+@import '../../../brand-globals/src/less/wells';
+
+//
```
Comment (jodiedoubleday) [inline brands/cardiff-airport/src/less/wells.less]: this file is probably not required

**Pull request merged**